extras/rya.export/export.client/pom.xml

```diff
@@ -46,6 +46,11 @@ under the License.
             <artifactId>rya.export.accumulo</artifactId>
             <version>3.2.10-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.rya</groupId>
+            <artifactId>rya.export.mongo</artifactId>
+            <version>3.2.10-SNAPSHOT</version>
```

Comment:
Remove the version here so it'll use the managed version instead.  Do the same for the ones above that are in the org.apache.rya groupId.